### ISSUE TITLE
x/pkgsite: <h3>Frequently asked questions:</h3> is first and only header on home page

### ISSUE THREAD

**Issue by jba** [golang/go#41039]
The first header on a page should be marked as `<h1>`. It would also be preferable if FAQ was not the first header on the page. Screen reader users jump to headers to skip repeated content in the page header/nav sections and, in this case, moving to the first header navigates users past the content of the page, straight into the footer.

**Comment by jamalc** [comment]
Fixed.
